test/JDBC/expected/BABEL_4442.out

```diff
@@ -0,0 +1,195 @@
+CREATE DATABASE db1
```

Comment:
Use names like `BABEL_4442_db1`, `BABEL_4442_s1` etc. like the pattern we follow in other tests.